### PR TITLE
feat: add wishlist component

### PR DESCRIPTION
**Pull request by Sahil25061999**

### CODE REVIEW

**Review by Anurag021 — APPROVED** [review]
code Looks good to me.

File: wishlist.html [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+        <h3>Socials</h3>
+        <ul class="list-style-none footer-list">
+          <li class="footer-list-items"><a href="#">Instagram</a></li>
+          <li class="footer-list-items"><a href="#">Twitter</a></li>
+          <li class="footer-list-items"><a href="#">Facebook</a></li>
+        </ul>
+      </section>
+    </footer>
+  </body>
+</html>
```
Comment (deekshithmd) [inline wishlist.html:176]: for me code looks clean and good to merge

**Review by deekshithmd — APPROVED** [review]

File: wishlist.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+.wishlist {
+  flex-wrap: wrap;
+}
+
+.card {
+  border: 2px solid var(--secondary-text-color);
+}
+
+.card-foot {
+  margin-bottom: 0;
```
Comment (parvez-rashid) [inline wishlist.css:29]: I don't understand why setting margin to 0 is required, it should be taken care of in CSS reset.

**Review by rahulk31 — APPROVED** [review]
Code structure looks clean to me and you have made great use of your Component Library which is impressive.

One suggestion though:
As mentioned on LMS the wishlisted product should have a remove product button. If you are going to implement that on the heart icon as remove button then its okay and can be merged.
Else i would suggest to add that button.
Reference:
image

**Review by parvez-rashid — APPROVED** [review]
UI Review: The UI looks great, clean and minimal. I like the scrollbar! As for the code, please have a look at the comment below, other than that, code can be merged.

File: wishlist.css (hunk 2 above)
Comment (Sahil25061999) [reply wishlist.css:29]: Its because i have given margin-bottom to card component in the component library.

**Pull request merged**